code/game/objects/items/fulton.dm

```diff
@@ -157,8 +161,8 @@ var/global/list/deployed_fultons = list()
 
 /obj/item/stack/fulton/proc/return_fulton(turf/return_turf)
 
-	// Fulton is not in space, it must have been collected.
-	if(!istype(get_area(attached_atom), /area/space/highalt))
+	// Fulton is not in reservation, it must have been collected.
+	if(!(get_turf(src) in reservation.reserved_turfs))
```

Comment:
Correct me if I'm wrong but I think you'd still want to delete the reservation then, no ?

Reply:
Ideally probably on pickup proper instead, in case something happens to the fulton object in the meantime

Reply:
oddly, /recall_fulton is the proc used when picking it up by a dropship